splash-screen/ios/Plugin/SplashScreen.swift

```diff
@@ -117,7 +117,7 @@ import Capacitor
     // Update the bounds for the splash image. This will also be called when
     // the parent view observers fire
     private func updateSplashImageBounds() {
-        var window: UIWindow? = UIApplication.shared.delegate?.window as? UIWindow
+        var window: UIWindow? = UIApplication.shared.delegate?.window
```

Comment:
Thanks for the PR, but the code doesn't compile, I think it should look like this:
```suggestion
        var window: UIWindow? = UIApplication.shared.delegate?.window ?? nil
```